wiki/Ranking_criteria/en.md

```diff
@@ -42,6 +42,7 @@ Important to understand before using:
 ### Guidelines
 
 - **Creators of the media used in a beatmap should be credited in the beatmap's description.** Per the [Content Usage Guidelines](/wiki/Rules/Content_usage_guidelines), media used in beatmaps should be permitted for use in osu!, and their licensing will likely detail how to credit their creators. Common reasons to omit credits from the description include exceeding difficulty in sourcing, creators' wishes to remain uncredited, and presence of credits in the artist, title, or source fields.
+  - Beatmaps using AI-generated backgrounds should instead credit them as such in the description.
```

Comment:
This is super unclear to the point of WTF.

The credit should go in the description instead of somewhere else? Bad sentence IMO.

But ignoring that, it is ambiguous in its aim. Need more context to even make a call on whether I even agree with this being added.

Reply:
This was discussed in the meeting held on the 7th of October. It was catalyzed by a [recent discussion regarding how AI backgrounds should be credited](https://osu.ppy.sh/beatmapsets/2066427/discussion/-/generalAll#/3966141).

Currently, the aim of this change is to explicitly mention how AI Backgrounds should be credited moving on, and the importance of the term "generated" being used when doing so.

Other things such as website tags were discussed during the meeting, but extra efforts would require the game to take a stance for AI content, so for now we're only trying to change the way these are credited as otherwise there is no clause that makes mentions of this.

Please let me know if you need to look at the meeting notes where this was discussed and ping you accordingly!

Reply:
Resolved with the most recent update.